cmd/fossa/main.go

```diff
@@ -125,6 +125,18 @@ func main() {
 				cli.BoolFlag{Name: "debug", Usage: debugUsage},
 			},
 		},
+		{
+			Name:   "report",
```

Comment:
I think we should allow for multiple report formats.

`fossa report dependencies` (default), `fossa report licenses`, `fossa report vulns`

Reply:
This is a really good idea.

Reply:
This is a good idea. I was thinking of report types as different kinds of notices to generate (e.g. NOTICE, ATTRIBUTION, LICENSE) but this makes a lot more sense. I'll implement this.